test/e2e/vault-decryption.spec.js

```diff
@@ -0,0 +1,115 @@
+const { strict: assert } = require('assert');
+const path = require('path');
+const fs = require('fs');
+const {
+  withFixtures,
+  WALLET_PASSWORD,
+  openSRPRevealQuiz,
+  completeSRPRevealQuiz,
+  tapAndHoldToRevealSRP,
+  completeCreateNewWalletOnboardingFlowWithOptout,
+} = require('./helpers');
+
+const VAULT_DECRYPTOR_PAGE = 'https://metamask.github.io/vault-decryptor';
+
+/**
+ * Retrieve the extension's storage file path that can be used
+ * with the Vault decryptor tool.
+ *
+ * Note that this folder is usually unavailable when running e2e tests
+ * on a test build, as test builds do not use the real browser storage.
+ *
+ * @param {WebDriver} driver
+ * @returns {Promise<string>} The SRP
+ */
+async function getExtensionStorageFilePath(driver) {
+  const { userDataDir } = (await driver.driver.getCapabilities()).get('chrome');
+  const extensionsStoragePath = path.resolve(
+    userDataDir,
+    'Default',
+    'Local Extension Settings',
+  );
+  // we expect the extension to have been installed only once
+  const extensionName = fs.readdirSync(extensionsStoragePath)[0];
+  const extensionStoragePath = path.resolve(
+    extensionsStoragePath,
+    extensionName,
+  );
+  const logFiles = fs
+    .readdirSync(extensionStoragePath)
+    .filter((filename) => filename.endsWith('.log'));
+
+  // Use the first of the `.log` files found
+  return path.resolve(extensionStoragePath, logFiles[0]);
+}
+
+/**
+ * Closes the announcements popover if present
+ *
+ * @param {WebDriver} driver
+ */
+async function closePopoverIfPresent(driver) {
+  const popoverButtonSelector = '[data-testid="popover-close"]';
+  try {
+    if (await driver.isElementPresent(popoverButtonSelector)) {
+      await driver.clickElement(popoverButtonSelector);
+    }
+  } catch (_err) {
+    // No popover
+  }
+}
+
+/**
+ * Obtain the SRP from the settings
+ *
+ * @param {WebDriver} driver
+ * @returns {Promise<string>} The SRP
+ */
+async function getSRP(driver) {
+  await openSRPRevealQuiz(driver);
+  await completeSRPRevealQuiz(driver);
+  await driver.fill('[data-testid="input-password"]', WALLET_PASSWORD);
+  await driver.press('[data-testid="input-password"]', driver.Key.ENTER);
+  await tapAndHoldToRevealSRP(driver);
+  return (await driver.findElement('[data-testid="srp_text"]')).getText();
+}
+
+describe('Vault Decryptor Page', function () {
+  it('is able to decrypt the vault using the vault-decryptor webapp', async function () {
+    await withFixtures(
+      {
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
```

Comment:
We don't really use test fixtures here (they wouldn't even work when this is run against the prod build), but `withFixtures` is quite handy to avoid dealing with all the network mocking and the webdriver config